src/ucc/api/ucc.h

```diff
@@ -1422,7 +1443,7 @@ typedef struct ucc_coll_op_args {
     ucc_coll_id_t                   tag; /*!< Used for ordering collectives */
     uint64_t                        root; /*!< Root endpoint for rooted
                                              collectives */
-} ucc_coll_op_args_t;
+} ucc_coll_args_t;
```

Comment:
seems ucc_memory_type_t not added as per PR description? BTW, should mem type should be aprt of coll_args (or)  buffer info?

Reply:
done